src/seedu/addressbook/parser/Parser.java

```diff
@@ -246,9 +246,11 @@ private Command prepareFind(String args) {
 
         // keywords delimited by whitespace
         final String[] keywords = matcher.group("keywords").split("\\s+");
+        //capitalise first letter every string in array keywords
+        for(int i=0 ; i<keywords.length ; i++)  {
+            keywords[i] = keywords[i].substring(0,1).toUpperCase() + keywords[i].substring(1).toLowerCase();
```

Comment:
I think there are string functions to do this in Java. Try to use them, in the interest of making code elegant.